sci-calculators/bc-gh/bc-gh-6.2.2.ebuild

```
@@ -0,0 +1,44 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Implementation of POSIX bc with GNU extensions"
+HOMEPAGE="https://git.yzena.com/gavin/bc"
```

```suggestion
HOMEPAGE="
	https://git.yzena.com/gavin/bc/
	https://github.com/gavinhoward/bc/
"
```

I see that you accept pull requests on GitHub, so let's make it easier to find it from `eix` output.